bundle/edu.gemini.auxfile.workflow/build.sbt

```diff
@@ -29,3 +29,5 @@ OsgiKeys.exportPackage := Seq(
   "edu.gemini.auxfile.client",
   "edu.gemini.auxfile.server",
   "edu.gemini.auxfile.copier")
+
+parallelExecution in Test := false
```

Comment:
This is completely unrelated, but (I think) works around spurious issues with test cases in this bundle that cannot be executed in parallel because they manipulate a shared `static` field in `edu.gemini.auxfile.server.file.FileManager`.